bugtool/cmd/root.go

```diff
@@ -315,6 +319,39 @@ func runAll(commands []string, cmdDir string, k8sPods []string) {
 			writeCmdToFile(cmdDir, cmd, k8sPods, enableMarkdown)
 			continue
 		}
+
+		if strings.Contains(cmd, "xfrm") {
+			var output []string
+			execResult, _ := execCommand(cmd)
+			// splitting the result as an array of string
+			lines := strings.Split(execResult, "\n")
+			// list of substrings to search for in lines
+			substrings := [...]string{"auth", "enc", "aead", "auth-trunc", "comp"}
+			for _, line := range lines {
+				for _, substr := range substrings {
+					// checking if the line contains our substring
+					matched, _ := regexp.MatchString(substr, line)
+					if matched {
+						// looking for hex substring which starts with '0x'
```

Comment:
I haven't checked whether the keys always have this format. Are we sure this is the case?

Reply:
https://gist.github.com/vishvananda/7094676. I was using this to add IPsec keys and all the keys were in hex format. But I will look into this.